permissions/Qr.permission

```diff
@@ -0,0 +1,11 @@
+# -*- mode: sh -*-
+
+# x-sailjail-translation-catalog = sailjail-permissions
+# x-sailjail-translation-key-description = permission-la-qr
+# x-sailjail-description = QR
+# x-sailjail-translation-key-long-description = permission-la-qr_description
+# x-sailjail-long-description = Recognize QR codes
```

Comment:
Recognize QR codes... QR code reading

Reply:
Indeed

Reply:
On Android camera permissions it is "Allow QR code reader" but that doesn't of the version and provider. We already dropped those "Allow" prefixes so please do not add that.

So my vote goes to "QR code reader" (not reading)

Reply:
See above ^